lib/matplotlib/figure.py

```diff
@@ -440,7 +456,10 @@ def _set_dpi(self, dpi, forward=True):
 
     def get_tight_layout(self):
         """
-        Return the Boolean flag, True to use :meth:`tight_layout` when drawing.
+        Returns
```

Comment:
I honestly think these cases are better handled by the single line form (I would write `Return whether tight_layout is applied after each draw.`).